atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -175,15 +178,15 @@ public int execute(final AtlasDbServices services) {
                         ? sweepRunner.dryRun(tableToSweep, batchConfig, startRow.get())
                         : sweepRunner.run(tableToSweep, batchConfig, startRow.get());
                 printer.info(
-                        "Swept from {} to {} in table {} in {} ms, examined {} cell values,"
-                                + " {}deleted {} stale versions of those cells.",
-                        encodeStartRow(startRow),
-                        encodeEndRow(results.getNextStartRow()),
-                        tableToSweep,
-                        watch.elapsed(TimeUnit.MILLISECONDS),
-                        results.getCellTsPairsExamined(),
-                        dryRun ? "would have " : "",
-                        results.getStaleValuesDeleted());
+                        "{} Swept from {} to {} in table {} in {} ms, examined {} cell values,"
+                                + (dryRun ? " would have" : "") + " deleted {} stale versions of those cells.",
```

Comment:
This will fail ErrorProne's `Slf4jConstantLogMessage` check, unfortunately 😞 

I think we can either use the original format, or just ignore my original comment :)

Reply:
I am going with pushing back on your comment and add a note to dry run saying that it will never delete any data.